src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/collection-select-overlay/main.js

```diff
@@ -77,59 +70,41 @@ define(function() {
         };
 
     return {
-        $columnNavigationContainer: '#child-column-navigation',
-        $columnNavigation: '#child-column-navigation .container',
-
         /**
-         * Initializes the collections list
+         * Initializes the overlay component
          */
         initialize: function() {
             // extend defaults with options
             this.options = this.sandbox.util.extend(true, {}, defaults, this.options);
 
             this.bindCustomEvents();
-            this.render();
+            this.renderOverlay();
```

Comment:
@nnatter can we call this 'openOverlay'. In the other overlay-component it was always 'openOverlay'